src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -661,6 +661,89 @@ class CombatLogParser extends MainCombatLogParser {
+        render: () => (
+          <div>
+            <div className="panel-heading">
+              <h2>Player Log Data</h2>
+            </div>
+            <div style={{ padding: '15px 22px 15px 15px' }}>
+              <div style={{ padding: '0px 22px 15px 0px' }}>Please use the below table to populate the Player Log section of the Mistweaver Spreadsheet by Garg.  <a href="http://www.peakofserenity.com/mistweaver/spreadsheet/" target="_blank" rel="noopener noreferrer">Link to the sheet</a><br /></div>
+              <div>
+              <table key='playlogdata-heading' style={{borderBottom: '1px solid #dddddd', borderTop: '1px solid #dddddd', align: 'left', padding: '8px', float: 'left', margin: '2px'}}>
+                <tr><td>Fight Length (Minutes)</td></tr>
+                <tr><td>Fight Length (Seconds)</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Mana Remaining</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Absorbless DTPS</td></tr>
+                <tr><td>Total Uplifting Trance procs</td></tr>
+                <tr><td>Unused UT %</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Thunder Focus Tea Casts</td></tr>
+                <tr><td>Effuse</td></tr>
+                <tr><td>Enveloping Mist</td></tr>
+                <tr><td>Essence Font</td></tr>
+                <tr><td>Renewing Mist</td></tr>
+                <tr><td>Vivify</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Mastery per EF</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Average SG Stacks</td></tr>
+                <tr><td>The Mists of Sheilun Procs</td></tr>
+                <tr><td>Dancing Mist Healing</td></tr>
+                <tr><td>Lifecycles-EnM</td></tr>
+                <tr><td>Lifecycles-Vivify</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>SotC Mana Return</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Mana Tea MP5</td></tr>
+                <tr><td>Misc MP5</td></tr>
+                <tr><td>Misc HPS</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Tier 20 2 Piece MP5</td></tr>
+                <tr style={{borderTop: '.5px solid #dddddd', borderBottom: '.5px solid #dddddd'}}><td>Tier 20 4 Piece Uptime</td></tr>
+              </table>
+              <table key='playlogdata-data' style={{borderBottom: '1px solid #dddddd', borderTop: '1px solid #dddddd', align: 'left', padding: '8px', float: 'left', margin: '2px'}}>
+                <tr key='Fight Length (Minutes)'><td>{Math.floor(this.fightDuration / 1000 / 60)}</td></tr>
```

Comment:
Specifying a `key` only does something when you put an element in a loop. Here it's unnecessary. I suggest removing it.